retrofit-converters/java8/src/test/java/retrofit/converter/java8/Java8OptionalConverterFactoryTest.java

```diff
@@ -31,57 +33,67 @@
 import retrofit2.Retrofit;
 import retrofit2.http.GET;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
 public final class Java8OptionalConverterFactoryTest {
   interface Service {
-    @GET("/") Call<Optional<Object>> optional();
-    @GET("/") Call<Object> object();
+    @GET("/")
+    Call<Optional<Object>> optional();
+
+    @GET("/")
+    Call<Object> object();
   }
 
   @Rule public final MockWebServer server = new MockWebServer();
 
   private Service service;
 
-  @Before public void setUp() {
-    Retrofit retrofit = new Retrofit.Builder()
-        .baseUrl(server.url("/"))
-        .addConverterFactory(Java8OptionalConverterFactory.create())
-        .addConverterFactory(new AlwaysNullConverterFactory())
-        .build();
+  @Before
+  public void setUp() {
+    Retrofit retrofit =
+        new Retrofit.Builder()
+            .baseUrl(server.url("/"))
+            .addConverterFactory(Java8OptionalConverterFactory.create())
+            .addConverterFactory(new AlwaysNullConverterFactory())
+            .build();
     service = retrofit.create(Service.class);
   }
 
-  @Test public void optional() throws IOException {
+  @Test
+  public void optional() throws IOException {
     server.enqueue(new MockResponse());
 
     Optional<Object> optional = service.optional().execute().body();
     assertThat(optional).isNotNull();
     assertThat(optional.isPresent()).isFalse();
   }
 
-  @Test public void onlyMatchesOptional() throws IOException {
+  @Test
+  public void onlyMatchesOptional() throws IOException {
     server.enqueue(new MockResponse());
 
     Object body = service.object().execute().body();
     assertThat(body).isNull();
   }
 
-  @Test public void delegates() throws IOException {
+  @Test
+  public void delegates() throws IOException {
     Object object = new Object();
-    Retrofit retrofit = new Retrofit.Builder()
-        .baseUrl(server.url("/"))
-        .addConverterFactory(new Converter.Factory() {
-          @Nullable @Override public Converter<ResponseBody, ?> responseBodyConverter(Type type,
-              Annotation[] annotations, Retrofit retrofit) {
-            if (getRawType(type) != Object.class) {
-              return null;
-            }
-            return value -> object;
-          }
-        })
-        .addConverterFactory(Java8OptionalConverterFactory.create())
-        .build();
+    Retrofit retrofit =
+        new Retrofit.Builder()
+            .baseUrl(server.url("/"))
+            .addConverterFactory(
+                new Converter.Factory() {
+                  @Nullable
```

Comment:
oooh I think this one is wrong

Reply:
weird that it’s different from the other `@Nullable` ones. Must retain annotation order strongly.